framework/include/fwk_status.h

```diff
@@ -81,6 +81,15 @@
 /*! Unrecoverable error */
 #define FWK_E_PANIC         -18
 
+/*!
+ * \brief Return a human readable string representation of a status code.
+ *
+ * \param status Status code value.
+ *
+ * \retval String representation of \p status
```

Comment:
This should be `\return` as we use `\retval` to describe individual return values.

Reply:
ok